wp-content/themes/the-world/functions.php

```diff
@@ -261,3 +261,8 @@ function tw_remove_core_block_patterns() {
 
 // Remove Windows Live Writer manifest link.
 remove_action( 'wp_head', 'wlwmanifest_link' );
+
+add_action('admin_menu', 'add_custom_menu_link');
+function add_custom_menu_link() {
```

Comment:
This function should be prefixed with `tw_` to prevent name collisions with other packages/plugins. Wrap the definition in `function_exists` check. See other action hook definitions in this file for structure. Include a comments for the function to describe what it is doing.